### DISCUSSION TITLE
lua_ls does not give any hints even with plain starter repo. Is this normal?

### DISCUSSION THREAD

**Discussion by tesarlukas** [NvChad/NvChad#2951] [Q&A] [answered]
As per title. This question come to me when my friend found weird I don't have any suggestions regarding the configuration of obsidian.nvim plugin (same thing happens with any other plugin). I always assumed this is a normal behaviour but then he showed me in his config he gets a lot of hints from LSP.

In the pictures below there is what lua_ls hints when configuring the plugin and also the fact that the LSP is not even aware of what require("osidian") is, even tho the plugin is downloaded and the code with setup function has been run without any errors.

![image](https://github.com/user-attachments/assets/17b824c0-e316-4356-9403-52bff53d7131)
![image](https://github.com/user-attachments/assets/cdc86582-3baa-4efe-ae4e-fcba1be41e00)
![image](https://github.com/user-attachments/assets/ef11447c-e1ca-43be-8b81-d0d277d39609)

**Comment by siduck** [comment, marked as answer]
its normal, either you need to include that plugin's types if it exists or create types or do nothing
